internal/routingtableentry.go

```diff
@@ -0,0 +1,143 @@
+/**
+ *  Copyright (c) 2020  Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package internal
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/json"
+	"fmt"
+)
+
+type (
+	// A Route represents an entry in the EARS routing table
+	Route struct {
```

Comment:
I wonder if we need to have a route Id in the DB. Eventually, I would think our rest API should support looking up or removing route based on a routeId even if we don't support it initially. It will be troublesome to update the schema later. We should have an routeId field right now (an auto-generated uuid or some unique id derived from hashes)

Reply:
ok